meetings/2022-10-26.md

```diff
@@ -72,6 +98,23 @@ URL: https://github.com/solid/specification/issues/462
 
 * SC: Would like to get a sense. Where to draw the line (for now). High-level view.
 * SC: Certainly technically possible. This particular discussion is not about how.
+* HS: I've implemented parts of Memento in [Reactive Solid](https://github.com/co-operating-systems/Reactive-SoLiD). There are it seems to me two parts to this problem: 1) the server needs to say it can support some of these features, and 2) a way for client to ask to set those features. This is similar to access control: A server can specify that it supports setting ACLs via a link header, allowing the client to set it by writing an access control rule. It seems we have the same here: we need some way for the server to say this container can do memento, and a way for the client then who wants to and is allowed to set that feature. This needs to be solved in a generic feature.
```

Comment:
```suggestion
* HS: I've implemented parts of Memento in [Reactive Solid](https://github.com/co-operating-systems/Reactive-SoLiD). It seems to me there are two parts to this problem: 1) the server needs to say it can support some of these features, and 2) a way for client to ask to set those features. This is similar to access control: A server can specify that it supports setting ACLs via a link header, allowing the client to set it by writing an access control rule. It seems we have the same here: we need some way for the server to say this container can do Memento, and a way for the client who wants to and is allowed to then set that feature. This needs to be solved in a generic feature.
```